pkg/cmd/task/describe.go

```diff
@@ -93,7 +93,8 @@ No params
  No steps
 {{- else }}
 {{- range $step := .Task.Spec.Steps }}
- {{decorate "bullet" $step.Name }}
+{{- $stepName := autoStepname $step.Name}}
+ {{decorate "bullet" $stepName}}
```

Comment:
nit: IIUC, this can be written as `{{ autoStepname $stepName | decorate "bullet" }} `

Reply:
good call thanks, 

(just read properly the [gotemplate](https://golang.org/pkg/text/template/) documentation this morning)